packages/clerk-js/src/ui/signUp/SignUpStart.tsx

```diff
@@ -102,7 +102,8 @@ function _SignUpStart(): JSX.Element {
   };
 
   React.useLayoutEffect(() => {
-    if (Object.values(fields).filter(f => f.enabled && !f.required).length) {
+    // Don't proceed with token flow if there are still optional fields to fill in
+    if (Object.values(fields).filter(f => f.show && !f.required).length) {
```

Comment:
🙃 
I think we can change this to `Array.prototype.some`

Reply:
Will do ✅